examples/debug/debug.sh

```diff
@@ -0,0 +1,145 @@
+    etcd_download_url: "{{ files_repo }}/github.com/etcd-io/etcd/releases/download/{{ etcd_version }}/etcd-{{ etcd_version }}-linux-{{ image_arch }}.tar.gz"
+    calicoctl_download_url: "{{ files_repo }}/github.com/projectcalico/calico/releases/download/{{ calico_ctl_version }}/calicoctl-linux-{{ image_arch }}"
+    calicoctl_alternate_download_url: "{{ files_repo }}/github.com/projectcalico/calicoctl/releases/download/{{ calico_ctl_version }}/calicoctl-linux-{{ image_arch }}"
+    calico_crds_download_url: "{{ files_repo }}/github.com/projectcalico/calico/archive/{{ calico_version }}.tar.gz"
+    flannel_cni_download_url: "{{ files_repo }}/kubernetes/flannel/{{ flannel_cni_version }}/flannel-{{ image_arch }}"
+    helm_download_url: "{{ files_repo }}/get.helm.sh/helm-{{ helm_version }}-linux-{{ image_arch }}.tar.gz"
+    crun_download_url: "{{ files_repo }}/github.com/containers/crun/releases/download/{{ crun_version }}/crun-{{ crun_version }}-linux-{{ image_arch }}"
+    kata_containers_download_url: "{{ files_repo }}/github.com/kata-containers/kata-containers/releases/download/{{ kata_containers_version }}/kata-static-{{ kata_containers_version }}-{{ ansible_architecture }}.tar.xz"
+    cri_dockerd_download_url: "{{ files_repo }}/github.com/Mirantis/cri-dockerd/releases/download/v{{ cri_dockerd_version }}/cri-dockerd-{{ cri_dockerd_version }}.{{ image_arch }}.tgz"
+    runc_download_url: "{{ files_repo }}/github.com/opencontainers/runc/releases/download/{{ runc_version }}/runc.{{ image_arch }}"
+    containerd_download_url: "{{ files_repo }}/github.com/containerd/containerd/releases/download/v{{ containerd_version }}/containerd-{{ containerd_version }}-linux-{{ image_arch }}.tar.gz"
+    nerdctl_download_url: "{{ files_repo }}/github.com/containerd/nerdctl/releases/download/v{{ nerdctl_version }}/nerdctl-{{ nerdctl_version }}-{{ ansible_system | lower }}-{{ image_arch }}.tar.gz"
+    gvisor_runsc_download_url: "{{ files_repo }}/storage.googleapis.com/gvisor/releases/release/{{ gvisor_version }}/{{ ansible_architecture }}/runsc"
+    gvisor_containerd_shim_runsc_download_url: "{{ files_repo }}/storage.googleapis.com/gvisor/releases/release/{{ gvisor_version }}/{{ ansible_architecture }}/containerd-shim-runsc-v1"
+ 
+---
+apiVersion: kubean.io/v1alpha1
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+  labels:
+    clusterName: ${CLUSTER_NAME}
+spec:
+  hostsConfRef:
+    namespace: kubean-system
+    name: ${CLUSTER_NAME}-hosts-conf
+  varsConfRef:
+    namespace: kubean-system
+    name:  ${CLUSTER_NAME}-vars-conf
+
+---
+apiVersion: kubean.io/v1alpha1
+kind: ClusterOperation
+metadata:
+  name: ${CLUSTER_NAME}-install-ops
+spec:
+  cluster: ${CLUSTER_NAME}
+  image: ghcr.m.daocloud.io/kubean-io/spray-job:${KUBEAN_VERSION}
+  actionType: playbook
+  action: cluster.yml
+  preHook:
```

Comment:
For Debug's purpose, is there no need to add Prehook and Posthook? `sleep infinity` may  be the main Action

Reply:
That's correct, the main effective action is `sleep infinity`, but users may also want to know the specific cluster LCM Ansible operation commands by viewing the `entrypoint.sh` script file.

Reply:
That could be comments on the top of scripts to indicate the usage.